src/components/select/select.scss

```diff
@@ -228,3 +228,41 @@ md-optgroup {
     border: 1px solid #fff;
   }
 }
+
+md-select-menu[multiple] {
+    md-option {
+        height: $select-option-height;
+        padding: 0 $select-option-padding 0 ($select-option-padding / 2);
+        padding-left: ($select-option-padding * 2) + ($select-option-padding / 2);
+
+        &:before {
```

Comment:
@DevVersion 

I believe that this can all be done using CSS and introducing the `md-checkbox` into the mix adds more overhead ,regarding watching values, than necessary.

But you are right, consistency is always appreciated, I will change this to mirror the `md-checkbox` css!

Muchos gracias for the feedback so far, more code to come.